src/login/logind-dbus.c

```diff
@@ -2038,6 +2042,68 @@ static usec_t nologin_timeout_usec(usec_t elapse) {
         return LESS_BY(elapse, 5 * USEC_PER_MINUTE);
 }
 
+int manager_load_scheduled_shutdown(Manager *m) {
+        _cleanup_fclose_ FILE *f = NULL;
+        int r;
+
+        f = fopen(SHUTDOWN_SCHEDULE_FILE, "r");
+        if (!f) {
+                if (errno == ENOENT)
+                        return 0;
+                return -errno;
```

Comment:
style suggestion: `return errno == ENOENT ? 0 : -errno;`
this would also allow you to remove the curly braces